Content.Shared/CCVar/CCVars.cs

```diff
@@ -18,10 +18,10 @@ public sealed class CCVars : CVars
             CVarDef.Create("server.id", "unknown_server_id", CVar.REPLICATED | CVar.SERVER);
 
         /// <summary>
-        ///     Name of the rules txt file in the "Resources/Server Info" dir. Include the extension.
+        ///     Path to the rules txt file in the "Resources/Server Info" dir. Include the extension.
         /// </summary>
-        public static readonly CVarDef<string> RulesFile =
-            CVarDef.Create("server.rules_file", "Rules.txt", CVar.REPLICATED | CVar.SERVER);
+        public static readonly CVarDef<string> RulesPath =
+            CVarDef.Create("server.rules_file", "/ServerInfo/Rules.txt", CVar.REPLICATED | CVar.SERVER);
```

Comment:
@deltanedas I just move /ServerInfo/ to CVar  , and if i understood ur example well it still be the same path with /ServerInfo/ as it supposed to be

Reply:
Oh , i understood , you're talking about values that goes from toml presets to this variables